teacher.rb

```diff
@@ -0,0 +1,12 @@
+class Teacher < Person
+  def initialize(age, name = 'unknown', parent_permission: ture, specialization: 'Physics')
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
+
+  attr_reader :specialization
+
+  def can_use_services?
+    @parent_permission = true
```

Comment:
can_use_services? method should return true and not set the `@parent_permission` to true. Kindly update this.

Reply:
Implemented 👍